### DISCUSSION TITLE
No me deja entrar en las categorias creadas por Joomla 4

### DISCUSSION THREAD

**Discussion by Haserzillo** [joomla/joomla-cms#33424] [Q&A]
Cuando intento entrar en una categoria que he creado en Joomla 4 me dice que no la encuentra osea ver la categoria puedo verla pero no me deja entrar para ver los articulos que estan dentro de esa categoria

**Comment by PhilETaylor** [comment]
> When I try to enter a category that I have created in Joomla 4 it tells me that it cannot find it, so I can see the category I can see it but it does not let me enter to see the articles that are within that category

**Comment by ChristineWk** [comment]
Go to content: categories. 
Published / Unpublished / Archived / Trashed Items >  Check it.
content: articles > Filter options > Select Category > Chose Category
Check e.g. 1 article from selected Category

Ir al contenido: categorías.
Elementos publicados / no publicados / archivados / en la papelera > Marcar.
contenido: artículos > Opciones de filtro> Seleccionar categoría > Elegir categoría
Compruebe, por ejemplo, 1 artículo de la categoría seleccionada

Other Check: content: articles: Chose 1 article and check, if correct category is marked:  
Otro Verificar: contenido: artículos: Elija 1 artículo y verifique si está marcada la categoría correcta. 

![screen shot 2021-04-29 at 14 23 01](https://issues.joomla.org/uploads/1/e3f7fbb01c1ffaa405d600a6e7b865f8.jpg) <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33404">issues.joomla.org/tracker/joomla-cms/33404</a>.</sub>

**Comment by ghost** [comment]
@Haserzillo which backend-language do you use?
